docs/_utils/preview

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
```

Comment:
why the duplication of `preview` and `preview.sh` ?

Reply:
``preview`` and ``preview.sh``  are adapted from the main docs repo. I imagine that the reason was to wrap the python script to keep consistency when calling the other available commands.

Reply:
o.k. if that coming from sphinx upstream, that's o.k. whatever their reason are...